doc/openapi-user-guide.md

```diff
@@ -303,3 +304,133 @@ oc create secret generic threescale-provider-account --from-literal=adminURL=htt
 * Default provider account in the same namespace 3scale deployment
 
 The operator will gather required credentials automatically for the default 3scale tenant (provider account) if 3scale installation is found in the same namespace as the custom resource.
+
+## AWS STS Support
+
+With the new support for STS authentication (Secure Token Service for  short-term, limited-privilege security credentials), 
+the secret generated by the Cloud Credential tooling looks differ from AIM (Identity and Access Management) secret.
+There are two new fields AWS_ROLE_ARN and AWS_WEB_IDENTITY_TOKEN_FILE are present instead of
+AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY.  
+Below are samples for STS and AIM secrets:
```

Comment:
```suggestion
Below are samples for STS and IAM secrets:
```

Reply:
Done + STS section was moved from openapi-user-guide.md instead to operator-user-guide.md.